src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsidentifyremandperiodsapi/relevantremand/controller/RelevantRemandController.kt

```diff
@@ -34,7 +34,7 @@ class RelevantRemandController(
   private val remandDecisionService: IdentifyRemandDecisionService,
 ) {
   @PostMapping(value = ["/{prisonerId}"])
-  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'MANAGE_DIGITAL_WARRANT')")
+  @PreAuthorize("hasRole('ADJUSTMENTS_MAINTAINER') and hasRole('RELEASE_DATES_CALCULATOR') and hasRole('REMAND_IDENTIFIER')")
```

Comment:
having multiple roles here combined with an AND looks odd, but I imagine it's been thought about

Reply:
Yea I thought that too. Joel wanted to enforce that the user had ALL the roles to access the service.